src/main/java/org/tron/common/runtime/RuntimeImpl.java

```diff
@@ -346,20 +403,29 @@ private void create()
 
       AccountCapsule creator = this.deposit
           .getAccount(newSmartContract.getOriginAddress().toByteArray());
+
+      if (RuntimeImpl.isNewVersion()) {  // TODO according to version
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this trailing comment on the previous empty line. ([squid:TrailingCommentCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ATrailingCommentCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1670&issues=AWbSdHUuSgN4uCclEWnR&open=AWbSdHUuSgN4uCclEWnR)

[ISSUE_KEY]: AWbSdHUuSgN4uCclEWnR